js/navigation/base.js

```diff
@@ -10,60 +10,64 @@ define([
 
 (function( $, undefined ) {
 
-	// existing base tag?
-	var baseElement = $( "head" ).children( "base" ),
+	var base,
+
+		// existing base tag?
+		baseElement = $( "head" ).children( "base" ),
+
+		// DEPRECATED as of 1.5.0 and will be removed in 1.6.0. As of 1.6.0 only
+		// base.dynamicBaseEnabled will be checked
+		getDynamicEnabled = function() {
+			if ( $.mobile.dynamicBaseEnabled !== undefined ) {
```

Comment:
Is there a reason for checking strictly for `undefined` here seems like `$.mobile.dynamicBaseEnabled` should be enough?